pennylane/capture/make_plxpr.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The make_plxpr function and helper methods"""
+
+from typing import Callable, Sequence, Union
+
+has_jax = True
+try:
+    import jax
+except ImportError:
+    has_jax = False
```

Comment:
Is this used anywhere?

Reply:
No, its just a copy-paste of our standard "attempt to import jax" block - but probably it should be, `make_plxpr` should either raise an error or return `None` if the user doesn't have `jax` installed 🤔 I'll add something to that effect!